modules/rosa_cluster/main.tf

```diff
@@ -23,7 +23,7 @@ resource "rhcs_cluster_rosa_classic" "rosa_sts_cluster" {
   replicas             = local.worker_node_replicas
   aws_account_id       = data.aws_caller_identity.current.account_id
   availability_zones   = var.availability_zones
-  tags                 = var.additional_tags
+  tags                 = merge(var.additional_tags, { cluster_name = var.cluster_name })
```

Comment:
Cluster_name, nicety or needed for something else?

Reply:
Needed to identify the load balancers of this cluster. We currently don't have a tag value that differentiates clusters, so this originally was pulling out every load balancer of any rosa cluster in the region. Not ideal.